apps/essimporter/converter.hpp

```diff
@@ -526,7 +526,11 @@ class ConvertJOUR : public Converter
 class ConvertGAME : public Converter
 {
 public:
-    ConvertGAME() : mHasGame(false) {}
+    ConvertGAME()
+        : mHasGame(false)
+        , mGame(GAME())
```

Comment:
What is the GAME() part for? Looks redundant to me.

Reply:
Coverity scan complains about uninitialised struct members. Maybe there is a better way to do it.

Reply:
What do you mean by "Fixing GAME"?

Reply:
The `GAME` (and `DATAstruct`) members are left uninitialized in the default constructor. You can change the struct to do something like
```
struct DATAstruct
  {
    int mFlags{0};
    int mX{0}, mY{0};
  };
```
to ensure the fields are constructed.

Reply:
Done.